docs/buildinfo.md

```diff
@@ -0,0 +1,141 @@
+# Build information
+
+> **Warning**
```

Comment:
we can delete this even in v0.11

Reply:
IMO, we should keep this around until we've actually removed it. I don't think there's any harm in just having it still here for people who might need docs on migrating away from it.